README.md

```diff
@@ -56,16 +59,16 @@ The following project are included in Terasoluna Global Framework Common Library
     </td>
   </tr>
   <tr>
-    <td>3</td><td>terasoluna-gfw-jpa</td><td>Maven dependency definition for using JPA</td><td>No</td><td><pre>* Recommended dependency definition</pre></td>
+    <td>4</td><td>terasoluna-gfw-mybatis3</td><td>Maven dependency definition for MyBatis3</td><td>No</td><td><pre>* Dependency definition for MyBatis3</pre></td>
   </tr>
   <tr>
-    <td>4</td><td>terasoluna-gfw-mybatis2</td><td>Maven dependency definition for using mybatis2</td><td>No</td><td><pre>* Recommended dependency definition</pre></td>
+    <td>5</td><td>terasoluna-gfw-jpa</td><td>Maven dependency definition for JPA</td><td>No</td><td><pre>* Dependency definition for JPA</pre></td>
   </tr>
   <tr>
-    <td>5</td><td>terasoluna-gfw-security-core</td><td>Maven dependency definition for using spring-security (other than web)</td><td>No</td><td><pre>* Recommended dependency definition</pre></td>
+    <td>6</td><td>terasoluna-gfw-security-core</td><td>Maven dependency definition for using spring-security (other than web)</td><td>No</td><td><pre>* Dependency definition for Spring Security (other than web)</pre></td>
   </tr>
   <tr>
-    <td>6</td>
+    <td>7</td>
     <td>terasoluna-gfw-security-web</td>
```

Comment:
Should be remove `* CSRF counter-measure` from Functionalities of `terasoluna-gfw-security-web` .